protocol/contracts/libraries/Minting/LibWellMinting.sol

```diff
@@ -172,49 +214,29 @@ library LibWellMinting {
             uint40(s.season.timestamp),
             pumps[0].data
         ) returns (uint[] memory twaReserves, bytes memory snapshot) {
-            IERC20[] memory tokens = IWell(well).tokens();
-            (
-                uint256[] memory ratios,
-                uint256 beanIndex,
-                bool success
-            ) = LibWell.getRatiosAndBeanIndex(tokens, block.timestamp.sub(s.season.timestamp));
-
-            // If the Bean reserve is less than the minimum, the minting oracle should be considered off.
-            if (twaReserves[beanIndex] < C.WELL_MINIMUM_BEAN_BALANCE) {
-                return (0, snapshot, new uint256[](0), new uint256[](0));
-            }
-
-            // If the USD Oracle oracle call fails, the minting oracle should be considered off.
-            if (!success) {
-                return (0, snapshot, twaReserves, new uint256[](0));
-            }
-
-            Call memory wellFunction = IWell(well).wellFunction();
-            // Delta B is the difference between the target Bean reserve at the peg price
-            // and the time weighted average Bean balance in the Well.
-            int256 deltaB = int256(IBeanstalkWellFunction(wellFunction.target).calcReserveAtRatioSwap(
-                twaReserves,
-                beanIndex,
-                ratios,
-                wellFunction.data
-            )).sub(int256(twaReserves[beanIndex]));
-
-            return (deltaB, snapshot, twaReserves, ratios);
+                                // well, reserves, snapshot  lookback
```

Comment:
can we add a comma here? also, †hink the comment should match the tab spacing of the block. If you want the text to be above the variables, perferable to indent
```suggestion
//                          well, reserves, snapshot  lookback
``` 
for example

Reply:
fixed here. removed the first one since it was rather obvious